velox/functions/sparksql/tests/ArithmeticTest.cpp

```diff
@@ -133,6 +133,38 @@ TEST_F(RemainderTest, int64) {
   EXPECT_EQ(-1, remainder<int64_t>(INT64_MIN, INT64_MAX));
 }
 
+TEST_F(RemainderTest, double) {
+  EXPECT_DOUBLE_EQ(0.0f, remainder<double>(2.0, 1.0).value());
```

Comment:
I wonder if we can add a lambda function returning the double result to replace `remainder<double>(a, b).value()` which repeats sevaral times.

Reply:
Done~